examples/hbase-snapshot-migration-scripts/export-snapshot.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/bash
+# This script creates and exports hbase snapshots to another cluster
+# Run this script on HBase 1x cluster
+# Usage: ./export-snapshot.sh <path/to/root/dir>
+
+# Parse the argument
+if [ -z "$1" ] || [ "$1" == '-h' ]; then
+  echo "Usage: $0 <path/to/hbase-rootdir>"
+  exit 1
+fi
+
+# Set current time
+time=$(date +'%Y%m%d-%H%M%S')
+
+# List all tables
+echo "list" | hbase shell -n | sed -e '1,/seconds/ d' |
+  while IFS='' read -r tableName || [[ -n "$tableName" ]]; do
+    if [[ $tableName =~ ^\s*$ ]]; then
+      continue
+    fi
+    # Create snapshot of each table
+    echo "snapshot '${tableName}', '${time}_${tableName}'" | hbase shell -n &&
+      echo "Snapshot created: ${time}_${tableName}"
+  done
+
+# Export snapshots to another cluster
+echo "list_snapshots" | hbase shell -n | sed '1,/seconds/ d' |
```

Comment:
Should we only move the snapshots we create?

If we ran this command twice, do we end up exporting the snapshots from both runs? Or if there are just other snapshots that happen to be on the system, would those be copied as well?

Maybe something like this would filter just down to the snapshots that were created:
```suggestion
echo "list_snapshots" | hbase shell -n | sed '1,/seconds/ d' | grep $time 
```

Although the structural change here could also be if we have the snapshot above (~like around line 23), then do we need to run list snapshots here? Could we export as we go? The extra validation that they show up in the list command isn't necessarily a bad thing though.

Reply:
adding `grep $time` makes sense. 

keeping creating and exporting snapshots separate adds more validation so this structure should be okay.